src/Themes/HackathonTheme/Views/Widget-VCard.liquid

```diff
@@ -0,0 +1,28 @@
+{% assign numberOfActions = Model.ContentItem.Content.Actions | size %}
+
+<v-card>
+  {% if Model.ContentItem.Content.VCard.Title.Text %}
+    <v-card-title>{{ Model.ContentItem.Content.VCard.Title.Text }}</v-card-title>
+  {% endif %}
+  {% if Model.ContentItem.Content.VCard.Subtitle.Text %}
+    <v-card-subtitle>{{ Model.ContentItem.Content.VCard.Subtitle.Text }}</v-card-subtitle>
+  {% endif %}
+  {% if Model.ContentItem.Content.VCard.Content.Html != "" %}
+    <v-card-text>
+      {{ Model.ContentItem.Content.VCard.Content.Html | sanitize_html | shortcode | raw }}
```

Comment:
Render the shape here as well

Reply:
Will it render the shortcodes correctly?